src/services/PaymentsService.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * PaymentsService
+ * Provide services related to Payments (Nodo) to communicate with PagoPa and Backend API
+ */
+
+import { Either, left, right } from "fp-ts/lib/Either";
+
+import { clients as pagoPaSoapClient } from "italia-pagopa-api";
+import {
+  InodoAttivaRPTInput,
+  InodoAttivaRPTOutput,
+  InodoVerificaRPTInput,
+  InodoVerificaRPTOutput
+} from "italia-pagopa-api/dist/wsdl-lib/PagamentiTelematiciPspNodoservice/PPTPort";
+import { PagoPaConfig } from "../Configuration";
+
+// TODO : [#158215861] Move it on PagoPaAPI
+const envelopeKey = "soapenv";
+
+// Send a request to PagoPa to check payment info
+export async function sendPaymentCheckRequestToPagoPa(
+  iNodoVerificaRPTInput: InodoVerificaRPTInput,
+  pagoPaConfig: PagoPaConfig
+): Promise<Either<Error, InodoVerificaRPTOutput>> {
+  try {
+    const pagamentiTelematiciPSPNodoClientBase = await pagoPaSoapClient.createPagamentiTelematiciPspNodoClient(
```

Comment:
creating a client on each request if very inefficient, what's the need?

we can create the client once and pass it to this function